spec/consensus/gossip/crdt.md

```diff
@@ -91,33 +91,33 @@ The tuple space is consulted through queries, which have the same form as the en
 Queries return all entries in their local views whose values match those in the query; `*` matches all values.
 For example, suppose a node's local view of the tuple space has the following entries, here organized as rows of a table for easier visualization:
 
-| Height | Round | Step      | Validator | Payload        |
-| ------ | ----- | --------- | --------- | -------------- |
-| 1      | 0     | Proposal  | v1        | pp1            |
-| 1      | 0     | PreVote   | v1        | vp1            |
-| 1      | 1     | PreCommit | v2        | cp1            |
-| 2      | 0     | Proposal  | v1        | pp2            |
-| 2      | 2     | PreVote   | v2        | vp2            |
-| 2      | 3     | PreCommit | v2        | cp2   [^equiv] |
-| 2      | 3     | PreCommit | v2        | cp2'  [^equiv] |
+| | Height | Round | Step      | Validator | Payload        | |
+|---| ------ | ----- | --------- | --------- | -------------- |---|
+$\lang$ | 1      | 0     | Proposal  | v1        | pp1            |$\rang$|
```

Comment:
I am afraid that Github does not render `\lang` and `\rang` properly. It renders a red `\lang` or `\rang` instead.